codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttConnectPayload.java

```diff
@@ -59,6 +77,10 @@ public String userName() {
     }
 
     public String password() {
```

Comment:
MQTT v3.1 requires string, MQTT v3.1.1 and above requires bytes. So technically, this can not be deprecated as long as v3.1 is around.